<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>520</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sahel</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="font">
            <font>
             <family>Sahel</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="accessibleName">
            <string>class1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPoet">
           <property name="accessibleName">
            <string>class1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="placeholderText">
            <string>شاعر یا نویسنده</string>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::VisualMoveStyle</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="accessibleName">
            <string>classFD</string>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="tabsClosable">
            <bool>true</bool>
           </property>
           <property name="movable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxDarkMode">
             <property name="accessibleName">
              <string>class2</string>
             </property>
             <property name="text">
              <string>حالت تاریک   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxBookmark">
             <property name="accessibleName">
              <string>class2</string>
             </property>
             <property name="text">
              <string>نشانه‌گذاری   </string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnPrevious">
             <property name="text">
              <string>قبل</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNext">
             <property name="text">
              <string>بعد</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewTab">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="accessibleName">
              <string>class2</string>
             </property>
             <property name="text">
              <string> برگ جدید </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnBookmarkForm">
             <property name="text">
              <string> صفحه نشانه‌ها </string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSearchForm">
             <property name="accessibleName">
              <string>class2</string>
             </property>
             <property name="text">
              <string> صفحه جست‌وجو </string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="font">
           <font>
            <family>Sahel FD</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="accessibleName">
           <string>classFD</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>350</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditSearch">
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>عبارت جست‌وجو</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarSearch">
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearch">
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="text">
             <string>  جست‌وجو  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="accessibleName">
             <string>class2</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdvancedSearch">
            <property name="text">
             <string>پیشرفته</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExportXML">
            <property name="text">
             <string>خروجی</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>lineEditPoet</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBoxDarkMode</tabstop>
  <tabstop>checkBoxBookmark</tabstop>
  <tabstop>btnPrevious</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>btnNewTab</tabstop>
  <tabstop>btnBookmarkForm</tabstop>
  <tabstop>btnSearchForm</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>lineEditSearch</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>btnAdvancedSearch</tabstop>
  <tabstop>btnExportXML</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
